templates/inspec/singular_resource.erb

```diff
@@ -15,13 +15,19 @@
 # frozen_string_literal: false
 
 <%= lines(autogen_notice :ruby) -%>
+<% vcr = ENV['VCR_MODE'] -%>
```

Comment:
Little confused by this.

It looks like we have two different sets of resources - one that work normally and one that only work with VCR cassettes. Is there a way of building VCR tests that work using the regular controls (so we don't have to build two versions that we assume work the same but may not)